src/endpoints/getMatch.ts

```diff
@@ -114,6 +114,15 @@ export const getMatch = (config: HLTVConfig) => async ({
       : undefined
   }))
 
+
+  maps.push({
+    // @ts-ignore
+    name: "All maps",
```

Comment:
I don't like messing with the maps array like that. I would be glad to have the id for all of the maps as a separate property on the match object, but not as a `maps` array element.